app/assets/javascripts/application.js

```diff
@@ -42,6 +42,8 @@
 //= require ./app/lib/rangy/rangy-textrange.js
 //= require ./app/lib/rangy/rangy-highlighter.js
 
+//= require froala_editor.min.js
+
```

Comment:
Since Zammad is open source and Froala Editor is not (https://www.froala.com/wysiwyg-editor/pricing) we can't use Froala Editor anyway.